src/tests/upvote.test.js

```diff
@@ -11,7 +11,7 @@ describe('UPVOTE CONTROLLER', () => {
   describe('PATCH UPVOTE INVITE', () => {
     it('it should upvote a post and increment upvote count', (done) => {
       chai.request(app)
-        .patch(`${upvoteBaseUrl}/90`)
+        .patch(`${upvoteBaseUrl}/1`)
```

Comment:
inviteID is not an integer... For security purpose use uuid as it's used in the Invite model and migration file.